packages/atomic/src/components/atomic-smart-snippet/atomic-smart-snippet-suggestions.tsx

```diff
@@ -16,9 +16,9 @@ import {
 } from '@coveo/headless';
 import {Hidden} from '../common/hidden';
 import {Heading} from '../common/heading';
-import {LinkWithResultAnalytics} from '../result-link/result-link';
 import {Button} from '../common/button';
 import {randomID} from '../../utils/utils';
+import {LinkWithResultAnalytics} from '../result-link/result-link';
```

Comment:
handling of that component is still in progress? or will it be in an upcoming PR?

Reply:
Benjamin will be handling it in an upcoming PR using the new atomic-smart-snippet-source component